### PR TITLE
feature: privilege mode for both cri manager and container manager

### PR DESCRIPTION
**Pull request by YaoZengzeng**
Signed-off-by: YaoZengzeng <yaozengzeng@zju.edu.cn>

### Ⅰ. Describe what this PR did

If the container is in privilege mode, it will have the full access to the host.

### Ⅱ. Does this pull request fix one issue?

fixes part of #635 

### Ⅳ. Describe how to verify it

```
root@pouch-ubuntu:/gopath/src/github.com/alibaba/pouch# pouch run busybox:1.26 brctl addbr foobar
brctl: bridge foobar: Operation not permitted
root@pouch-ubuntu:/gopath/src/github.com/alibaba/pouch# pouch run --privileged busybox:1.26 brctl addbr foobar
root@pouch-ubuntu:/gopath/src/github.com/alibaba/pouch# 
```

### Ⅴ. Special notes for reviews

As the development move on, we should consider if the container is in privilege mode in more places.

### CODE REVIEW

File: test/cli_start_test.go [hunk 1]
```diff
@@ -173,3 +173,12 @@ func (suite *PouchStartSuite) TestStartWithCapability(c *check.C) {
 	res.Assert(c, icmd.Success)
 	command.PouchRun("start", name).Assert(c, icmd.Success)
 }
+
+// TestStartWithPrivilege starts a container with privilege.
+func (suite *PouchStartSuite) TestStartWithPrivilege(c *check.C) {
+	name := "start-privilege"
+
+	res := command.PouchRun("create", "--name", name, "--privilege", busyboxImage, "brctl", "addbr", "foobar")
```
Comment (allencloud) [inline test/cli_start_test.go]: s/--privilege/--privileged

**Comment by YaoZengzeng** [comment]
@allencloud PTAL.

File: test/cli_create_test.go [hunk 2]
```diff
@@ -193,6 +193,25 @@ func (suite *PouchCreateSuite) TestCreateWithCapability(c *check.C) {
...
+	res.Assert(c, icmd.Success)
+
+	output := command.PouchRun("inspect", name).Stdout()
+
+	result := &types.ContainerJSON{}
+	if err := json.Unmarshal([]byte(output), result); err != nil {
+		c.Errorf("failed to decode inspect output: %v", err)
+	}
+
+	if !result.HostConfig.Privileged {
```
Comment (allencloud) [inline test/cli_create_test.go]: Here we can use assert to simplify the test case.

We usually use assert in test.

**Comment by YaoZengzeng** [comment]
@allencloud updated.

**Comment by allencloud** [comment]
LGTM

**Pull request merged**